woocommerce/woocommerce-gateway-paypal-express-checkout
# Add “Monochrome” and “Grayscale” as color options for Credit messaging

### Description

The "Graphic" layout for Credit messaging also supports options "Grayscale" and "Monochrome"

### Steps to test:


1. Check out this branch (`bnpl-add-color-options`).
1. Enable PayPal Credit and Credit messaging.
1. Select "Graphic" in the "Credit Messaging Layout" dropdown.
1. Choose the new "Monochrome" and "Grayscale" options and confirm they show up correctly on the frontend.
